.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,70 @@
+name: Build and push images to Docker Hub
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  build-and-push-strapi:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ secrets.DOCKERHUB_USERNAME }}/restore-strapi
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Build and push Strapi
+        uses: docker/build-push-action@v4
+        with:
+          context: "{{defaultContext}}:packages/strapi"
+          file: ./Dockerfile
+          #file: ./Dockerfile.prod
+          push: true
```

Comment:
None of these pushes have been updated

Reply:
Sorry could you clarify? I checked on hicsail docker hub and the 'pr-2' tag got pushed for both images, and they didn't overwrite 'latest' or generate any other tags, which I thought was the expected behavior since this PR is not yet merged